express-get-json/index.js

```diff
@@ -0,0 +1,37 @@
+const express = require('express');
+const app = express();
+
+const grades = {
+  12: {
+    id: 12,
+    name: 'Tim Berners-Lee',
+    course: 'HTML',
+    score: 95
+  },
+  47: {
+    id: 47,
+    name: 'Brendan Eich',
+    course: 'JavaScript',
+    score: 100
+  },
+  273: {
+    id: 273,
+    name: 'Forbes Lindsay',
+    course: 'JavaScript',
+    score: 92
+  }
+};
+
+const gradesArray = [];
+for (var o in grades) {
+  gradesArray.push(grades[o]);
+}
```

Comment:
This should be inside the middleware function, you want it to regenerate the array with every request so if the grades object changes the client will always receive the most current information.

Reply:
![image](https://user-images.githubusercontent.com/20393497/196265917-5b812839-1583-4eed-ac0b-e1347b138498.png)
moved.